File: Project/Remix_pwa/app/Models/implementations/ImageElement.ts
import { HtmlElement } from "../interfaces/HtmlElement";

export class ImageElement extends HtmlElement {
    override get Name(): string { return "img"; }

    constructor(src: string = "put your image url here", alt: string = "put alt text here") {
        super();
        this._attributes.set("src", src);
        this._attributes.set("alt", alt);
    }
}

File: Project/Blazor_pwa/Models/Implementations/ImageElement.cs
using Blazor_pwa.Models.Interfaces;

namespace Blazor_pwa.Models.Implementations
{
    public class ImageElement : HtmlElement
    {
        public override string Name => "img";

        public ImageElement(
            string src = "Введите адрес изображения",
            string alt = "Введите название изображения") : base()
        {
            Attributes.Add("src", src);
            Attributes.Add("alt", alt);
        }
    }
}